src/lib.rs

```diff
@@ -78,30 +78,43 @@ mod charutils;
 #[macro_use]
 mod macros;
 mod error;
+mod stringparse;
 mod numberparse;
 mod parsedjson;
 mod portability;
-mod stage1;
+
```

Comment:
TODO: this is so close, just need help based on how features should be defined in Cargo.toml